source/likely.js

```diff
@@ -2,5 +2,6 @@
 
 var likely = require('./index.js');
 
-window.likely = likely;
 window.addEventListener('load', likely.initiate);
```

Comment:
I’m not sure this will always work. The problem is that regardless of the way someone uses Likely code (by dropping a script into a page, with Webpack, or with Browserify), the script will execute _only_ in browser, where `module` will be undefined. We, however, want to make Likely auto-initialize only when it’s dropped as a `<script>`.